clientv3/integration/watch_test.go

```diff
@@ -320,8 +320,8 @@ func testWatchCancelRunning(t *testing.T, wctx *watchctx) {
 		select {
 		case <-time.After(time.Second):
 			t.Fatalf("took too long to close")
-		case v, ok = <-wctx.ch:
-			if ok {
+		case v, ok2 := <-wctx.ch:
```

Comment:
why change to ok2?

Reply:
Otherwise, we need another line to define `v` to not break `ineffassign`.